.gitignore

```diff
@@ -9,6 +9,7 @@ capybara-*.html
 /tmp
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/schema.rb
```

Comment:
good. I did that since it's generated and u guys suggested we ignore it last time. it seems like that's not a good idea after all.